selenium/javaScript/Chapter1:-nightwatch for sinlge browser/.gitignore

```diff
@@ -0,0 +1,5 @@
+//exclude below files/dir
+
+node_modules/*
+reports/*
+lambda_api.log
```

Comment:
what is this lambda_api.log file? why it is lambda?
Is this log file really created